### ISSUE TITLE
Representing CDA documents as FHIR documents

### ISSUE THREAD

**Issue by chengliu22** [synthetichealth/synthea#681]
Hi,
It's our understanding that a CDA document should be represented in FHIR as a bundle of type document with the first resource in the bundle being a composition. 
When we turn on the exporter.fhir.export, should the output be a bundle of type of document? Currently the output we get is a bundle of type of transaction.

**Comment by jawalonoski** [comment]
No, by default the output should be of type transaction, and not document.

Synthea has a property called `exporter.fhir.transaction_bundle` (defaults to `true`).

If `true` then Synthea exports FHIR as transaction bundles.
If `false` then Synthea exports FHIR as collection bundles.

Currently there is no support for document bundles.

Currently the CDA and FHIR exporters are completely independent.
